inc/api.php

```
@@ -184,40 +207,47 @@ function djz_get_gamipress($request) {
                 'level' => 1,
                 'rank' => 'Zen Novice',
                 'achievements' => [],
+                'pointsType' => 'zen-points',
             ],
         ];
         set_transient($cache_key, $fallback, DJZ_CACHE_GAMIPRESS);
         return rest_ensure_response($fallback);
     }
     
-    $points = (int)gamipress_get_user_points($user_id, 'zen-points');
+    $points_type = djz_get_gamipress_points_type_slug();
```

PHPMD (2.15.0)

The variable $points_type is not named in camelCase. (undefined)

(CamelCaseVariableName)